SetupAppveyor.ps1

```diff
@@ -0,0 +1,15 @@
+# only run if on appveyor, don't want to disable strong name verification elsewhere
+if ($env:APPVEYOR) {
+    # mount file share with test easy files
+    $drive = $env:TestEasySupportPath.replace("/", "");
+    echo "net use z: \\${env:FilesAccount}.file.core.windows.net\testeasy /u:${env:FilesAccount} <key>"
+    net use $drive \\${env:FilesAccount}.file.core.windows.net\testeasy /u:${env:FilesAccount} ${env:FilesKey}
+
+    # disable strong name verification
```

Comment:
This is only needed because the approach you've taken is to rebuild the binaries on the DB run, and they're delay signed. Another drawback there is that we're not running the tests against the actual signed packages we'll be deploying.

How hard would it be to have the SDK build push the requisite artifacts to the Files account you already have set up? That keeps the tests running the way that they've always run, except rather than a network share for the latest bits, its using Files.